pkg/store/tree.go

```diff
@@ -160,3 +151,29 @@ func addPathToTree(path, value string, nodeif *interface{}) error {
 
 	return nil
 }
+
+func handleLeafValue(nodemap map[string]interface{}, value *change.TypedValue, pathelems []string) {
+	switch value.Type {
+	case change.ValueTypeEMPTY:
+		// NOOP
+	case change.ValueTypeSTRING:
+		(nodemap)[pathelems[1]] = (*change.TypedString)(value).String()
+	case change.ValueTypeINT:
+		(nodemap)[pathelems[1]] = (*change.TypedInt64)(value).Int()
+	case change.ValueTypeUINT:
+		(nodemap)[pathelems[1]] = (*change.TypedUint64)(value).Uint()
+	case change.ValueTypeDECIMAL:
+		(nodemap)[pathelems[1]] = (*change.TypedDecimal64)(value).String()
+	case change.ValueTypeFLOAT:
+		(nodemap)[pathelems[1]] = (*change.TypedFloat)(value).Float32()
+	case change.ValueTypeBOOL:
+		(nodemap)[pathelems[1]] = (*change.TypedBool)(value).Bool()
+	case change.ValueTypeBYTES:
+		(nodemap)[pathelems[1]] = (*change.TypedBytes)(value).Bytes()
+	case change.ValueTypeLeafListSTRING:
+		(nodemap)[pathelems[1]] = (*change.TypedLeafListString)(value).List()
+	default:
+		(nodemap)[pathelems[1]] = fmt.Sprintf("unexpected %d", value.Type)
```

Comment:
I don't see the check for this error the calling method, is it fine or are we ignoring this ?

Reply:
the output will be visible in the JSON produced. It should never happen anyway all enum values are taken care of

Reply:
+1